src/debugging.js

```diff
@@ -63,6 +69,20 @@ export function addBidResponseHook(next, adUnitCode, bid) {
   next(adUnitCode, bid);
 }
 
+export function addBidRequestHook(next, bidRequest) {
+  const overrides = this;
+  if (Array.isArray(overrides.bidRequests)) {
+    const override = find(overrides.bidRequests, overrideBidRequest => (typeof overrideBidRequest.bidderCode === 'undefined' || overrideBidRequest.bidderCode === bidRequest.bidderCode));
+    if (override) {
+      Object.keys(override).filter(key => ['bidderCode', 'adUnitCode'].indexOf(key) === -1).forEach(key => {
+        bidRequest[key] = override[key];
+        utils.logInfo(`debug bidRequest override: ${key}=${override[key]}`);
```

Comment:
for consistency should use `logMessage()` which prefixes the message with "DEBUG:"

Reply:
also, would be nice to include the `adUnitCode` and `bidderCode` in the message so it's clear which request is affected, similar to the log message on line 63.